en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 
-You can use the `-` operator for number subtraction. The number values can come from a literal or a variable.
+    ```ballerina
+    number1 + 20;
+    ```
 
-#### Examples
+- Subtraction 
 
-- Subtracting integer literals .
+    You can use the `-` operator for number subtraction. 
 
-  ```ballerina
-  20 - 10
-  ```
-  
-- Subtracting decimal literals.
+    ###### Example:
 
-  ```ballerina
-  1.84 - 0.56  
-  ```
-  
-- Subtracting a literal from a numerical variable (`firstNumber`).
+    To subtract two integers, integer variable `yearOfBirth` from integer literal `2022`, you can use the following expression:
 
-  ```ballerina
-  firstNumber - 100
-  ```
-  
-- Subtracting a numerical variable (`secondNumber`) from another numerical variable (`firstNumber`).
+    ```ballerina
+    2022 - yearOfBirth;
+    ```
+ 
+- Division
 
-  ```ballerina
-  firstNumber - secondNumber
-  ```
+    You can use the `/` operator for number division. 
 
-### Number Division
+    ###### Example:
 
-You can use the `/` operator for number division. The number values can come from a literal or a variable.
+    To divide an integer variable `totalNumberOfStudents` from integer literal `5`, you can use the following expression:
 
-#### Examples
+    ```ballerina
+    totalNumberOfStudents / 5; 
+    ```
+ 
+- Multiplication
 
-- Dividing integer literals.
+    You can use the `*` operator for number multiplication. 
 
-  ```ballerina
-  20 / 10 
-  ```
-  
-- Dividing decimal literals.
+    ###### Example:
 
-  ```ballerina
-  1.84 / 0.56  
-  ```
-  
-- Dividing a numerical variable (`firstNumber`) by a literal.
+    To multiply two float constants `60` by `5`, you can use the following expression:
 
-  ```ballerina
-  firstNumber / 100
-  ```
-  
-- Dividing a numerical variable (`firstNumber`) by another numerical variable (`secondNumber`).
+    ```ballerina
+    60 * 5;
+    ```
 
-  ```ballerina
-  firstNumber / secondNumber
-  ```
+- Modulo 
 
-### Number Multiplication
+    You can use the  `%` operator to get the remainder of a division. 
 
-You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+    ###### Example:
 
-#### Examples
+    To get the remainder of the division between an integer variable `n` and an integer literal `3`, you can use the following expression:
+  
+    ```ballerina
+    (n % 3) > 2
+    ```
 
-- Multiplying integer literals.
+### Value/type operators
 
-  ```ballerina
-  10 * 20
-  ```
-  
-- Multiplying decimal literals.
+- Equality 
 
-  ```ballerina
-  0.56 * 1.84
-  ```
-  
-- Multiplying a numerical variable (`firstNumber`) by a literal.
+    You can use the equality operators to check the equality of [Ballerina's](https://ballerina.io/) basic type literals. These values can come from a literal or a variable. **Note** that the resulting value from these operators is always of `boolean` type.   
```

Comment:
Suggestion: "These values can be derived from a literal or a variable. The resulting value of these operators is always of the `boolean` type."

Reply:
If the part about the resulting value needs to be highlighted as a note, can we use "!!! note"? Having notes in two forms can be inconsistent.

Reply:
fixed.